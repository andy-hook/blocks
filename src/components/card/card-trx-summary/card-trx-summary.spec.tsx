import React from "react"
import CardTrxSummary from "./card-trx-summary"
import { render } from "@test-utils"

describe("<CardTrxSummary />", () => {
  it("renders correctly", () => {
    const tree = render(<CardTrxSummary transactions={""} />)
    expect(tree).toBeTruthy()
  })
})
