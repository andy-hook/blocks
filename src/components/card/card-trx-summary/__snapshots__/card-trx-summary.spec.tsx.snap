// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardTrxSummary /> renders correctly 1`] = `
<div
  className="sc-htpNat cjijsP"
>
  <ul
    className="sc-bxivhb gNgCgd"
  >
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH dIGGHs"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH dIGGHs"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
    <li
      className="sc-ifAKCX cFlEyZ"
    >
      <div
        className="sc-bdVaJa ldkUXY"
      >
        <div
          className="sc-bwzfXH jZccLu"
        />
      </div>
    </li>
  </ul>
</div>
`;
